applications/launchpad_v2/src/components/Tag/Tag.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react'
+import { ThemeProvider } from 'styled-components'
+
+import themes from '../../styles/themes'
+import Tag from './'
+import SVGCheck from '../../styles/Icons/Check'
+
+describe('Tag', () => {
+  it('should render Tag component without crashing', () => {
+    render(
+      <ThemeProvider theme={themes.light}>
+        <Tag icon={<SVGCheck />}>Testing</Tag>
+      </ThemeProvider>,
+    )
+
+    const el = screen.getByText('Testing')
+    expect(el).toBeInTheDocument()
+    expect(el).toHaveStyle('fontSize: 12')
```

Comment:
Checking the style props is not necessary a good thing. For instance, we may want to add new theme in the future and parametrise the tag's `fontSize` the same way as `color`, `background` or `borderRadius`, etc.. So I would recommend to not check style props, unless:

1. It is explicitly said in requirements that the specific element has to have a given style. But it's very rare case.
2. When we need to check whether something is visible to the user, but not how it looks. So it makes more sense to check props like `display` or `opacity`, but not colors, spacing etc.

So... I would just remove this: `expect(el).toHaveStyle('fontSize: 12')`, since this assertion: `expect(el).toBeInTheDocument()` seems to be good enough.

Reply:
👍